examples/gno.land/r/gnoland/monit/monit.gno

```diff
@@ -20,7 +20,7 @@ var (
 	lastUpdate       time.Time
 	lastCaller       std.Address
 	wd               = watchdog.Watchdog{Duration: 5 * time.Minute}
-	owner            = ownable.New() // TODO: replace with -> ownable.NewWithAddress...
+	OwnerStore       = ownable.New() // TODO: replace with -> ownable.NewWithAddress...
```

Comment:
I think there is a naming problem here. Why change this? `Owner` seems correct to me; it doesn't look like a store. Additionally, it's weird to end with `OwnerStore` when calling `ownable.New`. I suggest either renaming it to `ownable.NewStore` or changing the variable itself. However, based on what `New` actually returns, it doesn't make too much sense to refer to it as a store.

Reply:
Store was the best I can come up with; ideally you could just call it `ownable` but that conflicts with the pkgname. `Owner` conflicts with the ownable.owner field. it's kind of weird; and yes, `store` might not be the best naming.

Reply:
But Admin & Pause also don't ring the correct bells, not sure what to do.

Reply:
Switched to:

```go
var Ownable = ownable.New()
```

This seems optimal